### PR TITLE
Playbook Ansible

### PR DESCRIPTION
**Pull request by acmano7**
New playbook for automation.

### CODE REVIEW

File: Playbook.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    - name: Create Directory for Project
+      file:
+        path: /HelloWorldMaven
+        state: directory
+        mode: '0775'
+        recurse: yes
+        owner: root
+        group: root
+
+    - name: Clone App Repository
```
Comment (cilasbeltrame) [inline Playbook.yaml:56]: no need to clone the app, packer will do this automatically in the upcoming tasks. You can remove this block

File: Playbook.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    - name: Download and Unarchive Maven
+      unarchive:
+        src: https://dlcdn.apache.org/maven/maven-{{ maven_major_version }}/{{ maven_version }}/binaries/apache-maven-{{ maven_version }}-bin.tar.gz
+        dest: /usr/local/apache-maven
+        remote_src: yes
+
+    - name: Config ~/.bashrc
+      blockinfile:
+        create: yes
+        path: ~/.bashrc #/root/.bashrc
```
Comment (cilasbeltrame) [inline Playbook.yaml]: if this path means **/root/.bashrc** use the absolute path

File: Playbook.yaml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      blockinfile:
+        create: yes
+        path: /etc/systemd/system/java-hello-world.service
+        block: |
+          [Unit]
+          Description=Java App
+
+          [Service]
+          ExecStart=/bin/bash -c "while true; do java -cp /HelloWorldMaven/target/jb-hello-world-maven-0.2.0.jar hello.HelloWorld > /tmp/java_app.txt && sleep 2; done"
+
```
Comment (cilasbeltrame) [inline Playbook.yaml:95]: what about restarting and stopping the service?
If this grows much, consider using a script for each action

File: Playbook.yaml [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  
+  vars:
+    maven_major_version: 3
+    maven_version: 3.8.6
+  
+  tasks:
+    - name: Install Default Java JRE/JDK Pkg
+      apt:
+        pkg:
+        - default-jre
```
Comment (cilasbeltrame) [inline Playbook.yaml]: what java version is this? Java 8?11? It is a best practice to fix specific package versions, especially when it is the run time of the apps, like java, node etc

**Review by cilasbeltrame — APPROVED** [review]

File: Playbook.yaml (hunk 4 above)
Comment (acmano7) [reply Playbook.yaml]: It's Java 11. Suggestion has been applied.

File: Playbook.yaml (hunk 3 above)
Comment (acmano7) [reply Playbook.yaml:95]: This is a script, so stop and start are ok for now.

File: Playbook.yaml (hunk 2 above)
Comment (acmano7) [reply Playbook.yaml]: Suggestion has been applied.

File: Playbook.yaml (hunk 1 above)
Comment (acmano7) [reply Playbook.yaml:56]: We will keep it until packer task.

**Pull request merged**